Gemfile.lock

```diff
@@ -590,6 +592,3 @@ DEPENDENCIES
   warden (~> 1.2)
   warden-basic_auth (~> 0.2.1)
   will_paginate (~> 3.0)
-
-BUNDLED WITH
-   1.10.6
```

Comment:
@basiszwo Please update your bundler version. That this line is removed is an annoyance that the bundler team describes here:

bundler.io/blog/2015/06/24/version-1-10-released.html#bundled-with

**tl;dr:** New bundlers will add this line, old bundlers will remove it: **Git edit wars!** Update to a recent bundler: Worst case of one change per bundler version.